src/doc/build-script.md

```diff
@@ -80,8 +80,8 @@ Dependencies are declared through the `build-dependencies` section of the
 manifest.
 
 ```toml
-[build-dependencies.foo]
-git = "https://github.com/your-packages/foo"
+[build-dependencies]
+foo = { git = "https://github.com/your-packages/foo" }
```

Comment:
Yeah it's much nicer to see a list of these on one line rather than spread out across many. The other syntax is still accepted for super complicated dependencies, but I expect that'll be relatively rare compared to this.